docs/data-sources/virtual_environment_sdn_subnet.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: proxmox_virtual_environment_sdn_subnet
+parent: Data Sources
+subcategory: Virtual Environment
+description: |-
+  Retrieve details about a specific SDN Subnet in Proxmox VE.
+---
+
+# Data Source: proxmox_virtual_environment_sdn_subnet
+
+Retrieve details about a specific SDN Subnet in Proxmox VE.
+
```

Comment:
We should probably indicate here if the target resource does not exist in PVE, the datasource will return an error. It's pretty common (albeit a bad practice) to return a datasource with null attributes in this case, which some users may expect. 
It may also be worthwhile to mark the attributes as required as well, as I assume they all (or at least most of them) will be present on the ds.

Reply:
same for the other datasources

Reply:
> For me it seems clearer to return an error when the target resource doesn't exist as it can indicate the user something is wrong...

Yeah, I totally agree with that, and I think we should maintain this approach for all new data sources moving forward.